com.unity.ml-agents/Runtime/Sensors/Reflection/BoolReflectionSensor.cs

```diff
@@ -6,7 +6,7 @@ namespace Unity.MLAgents.Sensors.Reflection
     /// </summary>
     internal class BoolReflectionSensor : ReflectionSensorBase
     {
-        internal BoolReflectionSensor(ReflectionSensorInfo reflectionSensorInfo)
+        public BoolReflectionSensor(ReflectionSensorInfo reflectionSensorInfo)
```

Comment:
Had to make the constructors public to get Activator.CreateInstance cleanly. The classes are still internal, so this is not expanding the API.